index.html

```diff
@@ -134,12 +134,16 @@ <h2></h2>
           <p class="keywords"></p>
         </article>
       </template>
-      <nav>
-        <button id="previous-page-button"><</button>
-        <button id="next-page-button">></button>
-      </nav>
     </div>
   </main>
+  <div id="page-bottom">
+    <article id="placeholder-article" class="article">
+      <div class="placeholder-headline placeholder"></div>
```

Comment:
some of the elements have the placeholder class and some don't. I am not sure what the meaning of this class is. can you rename? can you make this clearer?